# false = mission item not reached, true = mission item reached
bool mission_item_reached
# false = mission item not accepted, true = mission item accepted
bool mav_mission_accepted
